src/Jeffijoe.MessageFormat.Tests/Jeffijoe.MessageFormat.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net5.0</TargetFramework>
```

Comment:
I would suggest multi-targeting the tests. This adds enough complexity that it may differ on .NET Framework vs .NET Core vs .NET 5.

Reply:
Good idea, is this easy to do?

Reply:
Yes, just change the test projects from using

```xml
<TargetFramework>net5.0</TargetFramework>
```

to

```xml
<TargetFrameworks>net5.0;netcoreapp3.1;net48</TargetFrameworks>
```

> **TIP:** Make sure you pay attention to the `s` in the name or you might be scratching your head for awhile.

In Visual Studio 2019, it will then show you all of the tests on each of the target frameworks and you can run them all or select the ones you want to run.

![image](https://user-images.githubusercontent.com/1538288/114055679-30cd4f80-98bb-11eb-8744-5b422160a356.png)


At the command line, it is a bit more complicated because AFAIK you have to run a `dotnet test` on each target framework as a separate command.

I believe it will work just to install the .NET 5.0 SDK (or at least you can always use the .NET 5 SDK as the entry point and all of the options it supports, but you may have to install other SDKs such as .NET Core 3.1 before installing .NET 5.0).

### Working Example

My setup is more complicated than yours because I test on multiple operating systems and use build assets to transfer the same binaries from the build agent to the test agents (you could just run `dotnet test` which would build from source on each OS, but I prefer to test the binaries I ship). I am also using Azure DevOps rather than GitHub Actions, but you should be able to set it up similarly there, too if you want to.

In my case, I do `dotnet publish` (in combination with properly setting the `<IsPublishable>` property to `true` for test projects and `false` for all of the others) to make the tests runnable on each OS and I specifically set up a build agent to test 1 target framework. But you are welcome to analyze the templates [here](https://github.com/NightOwl888/Morfologik.Stemming/blob/master/azure-pipelines.yml) and [here](https://github.com/NightOwl888/Morfologik.Stemming/tree/master/build/azure-templates) and use what you need (from anywhere in the project) to get it working.